### PR TITLE
Project - Method Edit and Delete photo

### PR DESCRIPTION
**Pull request by Melody-Le**
Done Edit Form

### CODE REVIEW

File: controllers/project_controller.js [hunk 1]
```diff
@@ -26,9 +26,25 @@ const controller = {
   },
 
   //Method GET: to Show form to edit profile:
-  editProject(req, res) {
+  async editProject(req, res) {
```
Comment (tlkiong) [inline controllers/project_controller.js]:
```suggestion
  async edit(req, res) {
```

File: controllers/project_controller.js [hunk 2]
```diff
@@ -26,9 +26,25 @@ const controller = {
...
     const projectId = req.params.project_id;
-    res.render("./projects/project_edit.ejs", { projectId });
+    const project = await Projects.findById(projectId);
+    res.render("./projects/project_edit.ejs", { project });
+  },
+
+  //Method PUT: to update project of specific ID
+  async updateProject(req, res) {
```
Comment (tlkiong) [inline controllers/project_controller.js]:
```suggestion
  async update(req, res) {
```

Comment (tlkiong) [reply controllers/project_controller.js]: Why need async when you don't have await?

**Review by tlkiong — APPROVED** [review]
LGTM aside from small comment 🚀

File: controllers/project_controller.js [hunk 3]
```diff
@@ -26,9 +26,47 @@ const controller = {
...
+  },
+
+  //Method PUT: to update project of specific ID
+  async update(req, res) {
+    const projectId = req.params.project_id;
+    const projectUpdate = req.body;
+    const newPhotos = projectUpdate.photos;
+    const newSkills = projectUpdate.skills;
+    const oldProject = await Projects.findById(projectId);
+    const oldProjectPhotos = oldProject.photos;
```
Comment (tlkiong) [inline controllers/project_controller.js:42]: Is photos array of url or what?

Comment (Melody-Le) [reply controllers/project_controller.js:42]: yes, photos is an arrays of URL photo

**Pull request merged**